common/common-api/src/main/java/org/ow2/proactive/scripting/Script.java

```diff
@@ -435,6 +429,17 @@ public boolean equals(Object o) {
         return false;
     }
 
+    /**
+     * Get MD5 hash value of the script without parameters
+     */
+    public static String digest(String script) {
+        try {
+            return new String(MessageDigest.getInstance("MD5").digest(script.getBytes()));
+        } catch (NoSuchAlgorithmException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1166)